### DISCUSSION TITLE
One backend (serverless) for many frontends

### DISCUSSION THREAD

**Discussion by Jancera** [vercel/community#3290] [Help] [answered]
### Question

I am thinking into a project that would have many frontends depending on customer preference with differents domains, so they are completely different projects, but the backend and the functionalities would be the same.

So I would like to have a set  of serverless functions that would be consumed by those frontends. Is it possible?

**Comment by amyegan** [comment, marked as answer]
Yes, you can have multiple frontends call the same set of serverless functions. You may need to [enable CORS](https://vercel.com/guides/how-to-enable-cors) to allow communication between the different domains.
